tests/unit/core/utils/test_i18n.js

```diff
@@ -37,6 +37,20 @@ describe(__filename, () => {
       expect(parts.afterLinkText).toEqual(' here.');
     });
 
+    it('lets you pass in additional variables', () => {
+      const anotherVar = 'another value';
+      const parts = getLocalizedTextWithLinkParts({
```

Comment:
Nit: Because of the number of tests that call it, it might be worth creating a `_getLocalizedTextWithLinkParts` function with some default values (e.g., `i18n = fakeI18n()`) to avoid some duplication in the tests.